vscode-antimony/src/server/bioservices_server/webservices.py

```diff
@@ -92,3 +114,55 @@ def annot_search_uniprot(self, query: str):
                 'prefix': 'uniprot',
             })
         return objects
+    
+    def annot_search_rhea(self, query: str):
+        self.init_rhea()
+        
+        if query.strip() == '':
+            return list()
+
+        try:
+            result_df = self.rhea.search(query, columns='rhea-id,equation')
+        except URLError:
+            raise NetworkError
+
+        result_df = result_df[0:20]
+        result_l = result_df.values.tolist()
+        objects = list()
+        for row in result_l:
+            id_ = row[0]
+            equation = row[1]
+            id_ = id_[5:]
+            objects.append({
+                'id': id_,
+                'name': equation,
+                'detail': id_,
+                'prefix': 'rhea',
+            })
+        return objects
+    
+    def annot_search_ontology(self, query: str, ontology_id: str):

```

Comment:
You should document your functions. Since you're using the typing package, you don't need to document parameters. However, you should describe what the function does.